### ISSUE TITLE
tests: composer-cli compose metadata fails

### ISSUE THREAD

**Issue by atodorov** [osbuild/osbuild-composer#643]
```
# composer-cli compose list
0396b0be-410b-44b6-97dd-e317f2b17c52 FINISHED test-blueprint 0.0.2 AWS

# composer-cli --json compose metadata 0396b0be-410b-44b6-97dd-e317f2b17c52
No Content-Disposition header; cannot get filename
```
exit code 1.

This causes the newly enabled sanity tests in #642 to fail.

**Comment by bcl** [comment]
osbuild-composer doesn't have the same concept of metadata that lorax-composer did, so this has not been implemented yet, and may not be added. The use case for metadata was that you had everything you needed to recreate a build, it was all of the files from the result directory except for the actual image. I'm not sure this makes sense anymore, and was likely only ever used for debugging things.

**Comment by atodorov** [comment]
@bcl If we decide that metadata doesn't make sense for osbuild-composer or that it is not something we care about we should think along the lines of disabling these commands (probably on the cli side) depending on backend. Otherwise people will run into them like I did and wonder why they fail.

**Comment by teg** [comment]
I think we should return the manifests, they have exactly the behavior @bcl describes.

**Comment by bcl** [comment]
Ah! Great, I didn't realize there was similar data available.

**Comment by bcl** [comment]
Fixed by #661
